### PR TITLE
Add id to example responses

### PR DESCRIPTION
**Pull request by me**
Clients should be allowed to rely on `id` being present in the response.

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.1.1
\ No newline at end of file
+1.0.0
```
Comment (jphastings) [inline VERSION:1]: 🎉

File: experiments/a_b_guid.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
+  "id": "feature_1",
```
Comment (jphastings) [inline experiments/a_b_guid.json]: For these to be representative of how other systems would work, I think the `id` of these experiments would probably need to be the file path, ie. `experiments/a_b_guid.json`

**Review by jphastings — APPROVED** [review]
Generally LGTM, though I think you'd need to make a few changes still to be really representative

File: experiments/a_b_guid.json (hunk 2 above)
Comment (me) [reply experiments/a_b_guid.json]: Ah, makes sense!

**Pull request merged**